flathunter/filter.py

```diff
@@ -123,6 +123,21 @@ def is_interesting(self, expose):
         if not found_objects:
             return True
         return False
+        
+class PPSFilter:
+    """Exclude exposes above a given price per square"""
+
+    def __init__(self, max_pps):
+        self.max_pps = max_pps
+
+    def is_interesting(self, expose):
+        """True if price per square is below max price per square"""
+        size = ExposeHelper.get_size(expose)
+        price = ExposeHelper.get_price(expose)
+        pps = price / size
```

Comment:
If size or price is None, this division is going to fail. Can you move the check above?

Reply:
Done.

Reply:
Great!